src/libstore/gc.cc

```diff
@@ -37,6 +37,7 @@ namespace nix {
 static std::string gcSocketPath = "/gc-socket/socket";
 static std::string gcRootsDir = "gcroots";
 
+namespace fs { using namespace std::filesystem; }
```

Comment:
I really don't get the need for importing `std::filesystem` into the `nix::fs` namespace. I mean, why not put all of `std` in the `nix` namespace...